observations/snow/sfcsno.yaml.j2

```diff
@@ -0,0 +1,112 @@
+-
+
+  # Observation Space (I/O)
+  # -----------------------
+  obs space:
+    name: sfcsno
+    obsdatain:
+      engine:
+        type: bufr
+        obsfile: "{{snow_obsdatain_path}}/{{snow_obsdatain_prefix}}sfcsno.tm00.bufr_d"
+        mapping file: "{{snow_obsdatain_path}}/bufr_sfcsno_mapping.yaml"
+    obsdataout:
+      engine:
+        type: H5File
+        obsfile: "{{snow_obsdataout_path}}/{{snow_obsdataout_prefix}}sfcsno{{snow_obsdataout_suffix}}"
+    simulated variables: [totalSnowDepth]
+  #
+
+  # Observation Operator
+  # --------------------
+  obs operator:
+    name: Composite
+    components:
+    - name: Identity
+    - name: BackgroundErrorIdentity
+  linear obs operator:
+    name: Identity
+  #
+
+  # Observation Filters (QC)
+  # ------------------------
+  obs pre filters:
+    - filter: Perform Action
+      filter variables:
+      - name: totalSnowDepth
+      action:
+        name: assign error
+        error parameter: 40.0
+    - filter: Variable Assignment
+      assignments:
+      - name: GrossErrorProbability/totalSnowDepth
+        type: float
+        value: 0.02
+      - name: BkgError/totalSnowDepth_background_error
+        type: float
+        value: 30.0
+  obs prior filters:
+    - filter: Bounds Check
+      filter variables:
+      - name: totalSnowDepth
+      minvalue: 0.0
+      maxvalue: 20000.0
+      action:
+        name: reject
+    - filter: Domain Check
+      where:
+      - variable:
+          name: MetaData/stationElevation
+        minvalue: -999.0
```

Comment:
This is for GTS yaml I guess. If so, the MetaData/stationElevation min and max value had been modified in the offline workflow. Suggest confirming reusing these min/max values for the GTS obs. used in the g-w.

Reply:
Good catch.